contents/newsletter/failure-modes-of-a-team-lead.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Failure modes of an engineering team lead 
+date: 2025-06-18
+author:
+  - lior-neu-ner
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/cec9bd6f50e59b2250958054823e5ce7_c967d62274.jpg
+featuredImageType: full
+tags:
+  - Product engineers
+  - Culture
+crosspost:
+  - Founders
+  - Blog
+---
+
+![](https://res.cloudinary.com/dmukukwp6/image/upload/cec9bd6f50e59b2250958054823e5ce7_c967d62274.jpg)
```

Comment:
I think this is a good image, but got to think about how it will fit as a thumbnail

Reply:
Yeah, this probably won't work as a thumbnail, but would be good to do a hoggy version of this and put in the body somewhere.